llvm/cmake/modules/VersionFromVCS.cmake

```diff
@@ -39,8 +39,14 @@ function(get_source_info path revision repository)
         OUTPUT_VARIABLE git_output
         ERROR_QUIET)
       if(git_result EQUAL 0)
-        string(STRIP "${git_output}" git_output)
-        set(${repository} ${git_output} PARENT_SCOPE)
+        # Avoid exposing sensitive data, e.g. usernames, passwords and
+        # private URLs.
+        string(FIND "${git_output}" "github.com/llvm/llvm-project" git_upstream)
```

Comment:
Depending on the protocol used, the URL might have a different format. For example, if using the Git protocol it's going to be `git@github.com:llvm/llvm-project.git`.

Reply:
Fixed in my latest update.